src/components/GaugeWeight.tsx

```diff
@@ -26,10 +26,11 @@ export default function GaugeWeight({
   const theme = useTheme()
 
   const gaugesInfo = Object.values(gauges)
-    .map(({ gaugeName, gaugeRelativeWeight, poolAddress }) => {
+    .map(({ gaugeName, gaugeRelativeWeight, poolAddress, isKilled }) => {
       const pool = Object.values(basicPools || {}).find(
```

Comment:
I think we can move the `isKilled` up here or we can just do a `.filter` before or after the `.map`